cumulusci/core/tests/utils.py

```diff
@@ -35,7 +35,7 @@ def emit(self, record):
             self.release()
 
     def reset(self):
-        """Reset the handler in TestCase.setUp() to clear the msg list"""
+        """Reset the handler in TestCase.setup_method() to clear the msg list"""
```

Comment:
Doesn't need to be in this PR, but we can replace MockLoggingHandler entirely with pytest's caplog fixture